### PR TITLE
OKTA-277747 Improve Runscope instructions

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?** 
Improved Runscope test instructions to explain what dummy variables are created before and during the SPEC test phase.
- **Is this PR related to a Monolith release?**
No

### Resolves:

* [OKTA-277747](https://oktainc.atlassian.net/browse/OKTA-277747)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md [hunk 1]
```diff
@@ -26,47 +26,77 @@ To get started using Runscope to test your SCIM API:
...
+
+## Sharing test results from Runscope
+
+As you refine your SCIM implementation, you can share API test results with your teammates or with people outside of your organization:
+
+1. From your Runscope dashboard, open the test result that you want to share.
+2. At the top of the test result, Change the **Private | Shareable** toggle from **Private** to **Shareable**.
+3. Copy the URL for the test result, it will look something like this:
+    `https://www.runscope.com/radar/abcdefghijkl/m01nopq2-3456-7r8s-9012-t34567uvw890/history/123ef4gh-i567-89j0-1k2l-3m4n5o678901`
+The test results can be viewed in detail, but the test can't be edited or be re-run by people outside of your team.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md:100]:
```suggestion
The test results can be viewed in detail, but the test can't be edited or re-run by people outside of your team.
```

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md [hunk 2]
```diff
@@ -26,47 +26,77 @@ To get started using Runscope to test your SCIM API:
...
-### Customize the imported Runscope test for your SCIM integration
-
-After importing the Okta SCIM test suite into Runscope, you need to configure the test for your SCIM integration:
-
-1. You should be looking at the **API Tests** screen in Runscope, if not, click the **Tests** tab at the top of the Runscope user interface.
+1. You should be looking at the **API Tests** window in Runscope, if not, click the **Tests** tab at the top of the Runscope user interface.
 1. You should see a test named **Okta SCIM X.X Spec Test** (where X.X corresponds to the SCIM version of the JSON file you uploaded).
 1. Click **Edit** on the test panel.
-1. The **Okta SCIM X.X SPEC Test** screen appears. Under the **Environment** section of your test, there is a collapsed section labelled **Test Settings** . Click the small arrow to expand this section.
+1. The **Okta SCIM X.X SPEC Test** window appears. Under the **Environment** section of your test, there is a collapsed section labelled **Test Settings** . Click the small arrow to expand this section.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md:38]:
```suggestion
1. The **Okta SCIM X.X SPEC Test** window appears. Under the **Environment** section of your test, there is a collapsed section labelled **Test Settings**. Click the small arrow to expand this section.
```

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md [hunk 3]
```diff
@@ -26,47 +26,77 @@ To get started using Runscope to test your SCIM API:
...
-    | SCIMBaseURL                    | `https://example.com/scim/v2`  `https://example.com/scim/v1` | For example, if your SCIM integration is hosted on <https://example.com> and uses a prefix of /scim/v2 then the *SCIMBaseURL* value for your integration would be: `https://example.com/scim/v2`. |
-    | auth                           | Bearer abcxyz1234567890     | Basic or OAuth authorization token                                                                                                                                                          |
+    | Variable Name | Example Values | Notes |
+    |:-|:-|:-|
+    | SCIMBaseURL | `https://example.com/scim/v2`  `https://example.com/scim/v1` | For example, if your SCIM integration is hosted on <https://example.com> and uses a prefix of /scim/v2 then the *SCIMBaseURL* value for your integration would be: `https://example.com/scim/v2`. |
+    | auth | Bearer abcxyz1234567890 | Basic or OAuth authorization token. |
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md:47]: I'm not sure what this should be, but AFAIK `Basic authentication` doesn't use tokens, so "Basic authorization token" is a misnomer. In contrast to Basic Auth, OAuth tokens are Bearer tokens.

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md [hunk 4]
```diff
@@ -26,47 +26,77 @@ To get started using Runscope to test your SCIM API:
...
+
+    During the test phase, several additional variables are created, based on results returned from your SCIM server.
+
+    | Variable Name | Example Value | Notes |
+    |:-|:-|:-|
+    | ISVUserid | `29cb3ed0-b0da-498a-ba0c-f146f15a65d5` | The first UUID returned from a GET request to your `/Users` endpoint. |
+    | idUserOne | `323da8f8-21b8-4b25-8322-90673d9e1bc7` | A UUID of a test user created on your SCIM server. |
+    | randomUserEmail | `Runscope300Hfluaklab151@atko.com` | An email address returned from your SCIM server. |
+
+1. To see the details of tests, click the name of each particular test case to expand the section. The details have information on the **Request**, **Response**, and **Connection** for each HTTP request of the test. Each test sends a composed GET or POST request to your SCIM server, and the HTTP status received in response determines the success or failure of each test.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md:87]:
```suggestion
1. To see the details of tests, click the name of each particular test case to expand the section. The details have information on the **Request**, **Response**, and **Connection** for each HTTP request involved in the test. Each test sends a composed GET or POST request to your SCIM server, and the HTTP status received in response determines the success or failure of each test.
```

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md (hunk 3 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md:47]: Would "OAuth Bearer token, or Basic authentication code" fix the description?

Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md]: Changed to 

> OAuth 2.0 Bearer token or Basic authentication code.

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md [hunk 5]
```diff
@@ -26,47 +26,77 @@ To get started using Runscope to test your SCIM API:
...
+  
+    This script sets up a number of randomly generated "dummy" variables that are used during the testing process:
+
+    | Variable Name | Example Value | Notes |
+    |:-|:-|:-|
+    | InvalidUserEmail | `abcdefgh@atko.com` | A specific email address considered invalid by the test. |
+    | randomEmail | `Runscope300Hfluaklab151@atko.com` | A valid email address. |
+    | randomFamilyName | `Hfluaklab151` | A valid last name. |
+    | randomGivenName | `Runscope300` | A valid first name. |
+    | randomUsername | `Runscope300Hfluaklab151@atko.com` | An valid user name. |
```
Comment (ykrestyanov-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md:62]: I guess it should be "A valid user name."

Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md]: Good catch - thanks!

**Pull request merged**